### ISSUE TITLE
SA1008 false positive before comment

### ISSUE THREAD

**Issue by lucky-ly** [DotNetAnalyzers/StyleCopAnalyzers#3945]
When the opening parenthesis is followed by a comment, adding a whitespace before the comment triggers SA1008.
I think, this should be allowed.

Some code to illistrateL
```csharp
var query = query.Where(x => someCondition && // this comment is fine
    otherCondition &&// and this is not fine, it triggers SA1003 which is good
    !(// but this is somehow fine too
        (otherCondition1) &&
        (otherCondition2) &&
        (otherCondition3)
    ) &&
    !( // and this comment triggers SA1008 because there is a whitespace
        (otherCondition4) ||
        (otherCondition5) ||
        (otherCondition6)
    ));
```
